.azure-devops/templates/smoke-tests.yml

```diff
@@ -0,0 +1,38 @@
+parameters:
+- name: pythonVersion
+  type: string
+  default: '3.8.x'
+- name: architecture
+  type: string
+  default: 'x64'
+- name: azureCLIVersion
+  type: string
+  default: released
+  values:
+  - min
+  - released
+  - msi
+
+steps:
+  - template: setup-dev-test-env.yml
```

Comment:
The problem with using this template is it includes installing dev components and changes in the env specific for CI runs executing the CLI tests, the smoke test runs should have minimal additional dev setup.